backend/Repositories/PlayersRepository.cs

```diff
@@ -14,36 +12,34 @@ public interface IPlayersRepository
         void Delete(string code);
     }
 
-    public class PlayersRepository: IPlayersRepository
+    public class PlayersRepository : IPlayersRepository
     {
         public List<Player> GetAll()
         {
-            return this.data;
```

Comment:
`this` is optional in C#.

class User {
  string name;

  void doSomething1() {
    var name = "x"; // you will also typically get warnings saying that `name` will hide `this.name`.
    // this.name != name;
  }
}